source/documentation/tools/jupyterlab.md

```diff
@@ -0,0 +1,88 @@
+# JupyterLab
+
+A development environment for writing Python code including Python notebooks
+
+See: <https://github.com/RobinL/cheatsheets_etc/blob/master/jupyter_conda.md>
+
+## Run notebooks
+
+In Jupyter, before you can successfully run the notebook, you'll need to select the Jupyter kernel for this project. If it doesn't appear in the drop-down list, run this in a terminal:
+
+```bash
+. myproject/venv/bin/activate
+python3 -m ipykernel install --user --name="venv" --display-name="My project (Python3)"
+```
+
+And if your project has analytical scripts that run in a terminal you could add:
```

Comment:
Does this belong in the Run Scripts section below?

Reply:
This was a copy and paste from the previous version, but I'm happy to change this.